src/save.rs

```diff
@@ -5,6 +5,7 @@ use std::hash::{Hash, Hasher};
 use std::io::Read;
 
 use byteorder::{LittleEndian, ReadBytesExt};
+use chrono::{DateTime, Utc};
```

Comment:
Can we turn this into a `pub use` so that dependents don't have to add `chrono` to their `Cargo.toml` to deal with these types?